scheduler/drilling-wits-depth-median/tests/test_e2e.py

```diff
@@ -0,0 +1,2 @@
+def test_e2e():
```

Comment:
cant write this test yet, because new version of sdk with split scheduled events was not deployed yet

Reply:
ltf doesnt support depth scheduled events yet